spring-pulsar/build.gradle

```diff
@@ -19,11 +18,13 @@ dependencies {
 	implementation 'com.fasterxml.jackson.core:jackson-core'
 	implementation 'com.fasterxml.jackson.core:jackson-databind'
 	implementation 'com.google.code.findbugs:jsr305'
+	compileOnly 'org.apache.pulsar:pulsar-client-reactive-adapter'
```

Comment:
We can go back to `api` when we change the COMB in auto-config to the reactive client rather than reactive template class.

Reply:
The idea was to not have spring-pulsar pull the reactive client. But that was maybe a bad idea. Setting it to `api.`

Reply:
@cbornet sorry, I meant I was able to put it to `optional` w/ this change. Not `api`.